packages/app/src/elements/login-signup.ts

```diff
@@ -460,7 +460,7 @@ export class LoginOrSignup extends StartForm {
                     this._accountExists();
                     return;
                 default:
-                    alert(e.message, { type: "warning" });
+                    alert(e.message || $l("Server error."), { type: "warning" });
```

Comment:
I added this default message because an encoding error has no message, and it seemed out of scope to implement a way to return a message on `validate()` (and I'm not even sure if you'd want to). We're not expecting people to hit that anyway, since the client will have validated the data, but still.